ouroboros-consensus/test-consensus/Test/Dynamic/Network.hs

```diff
@@ -98,7 +99,10 @@ createNetworkInterface
     -> peer                 -- ^ our peer
     -> NetworkApplication m peer
         (AnyMessage (ChainSync (Header blk) (Point blk)))
-        (AnyMessage (BlockFetch blk)) ()
+        (AnyMessage (BlockFetch blk))
+        (AnyMessage (ChainSync blk (Point blk)))
+        (AnyMessage (LocalTxSubmission (GenTx blk) String))
+        ()
     -> NetworkInterface m peer
 createNetworkInterface chans nodeIds us
```

Comment:
Maybe add a comment stating that we don't need to set up the two node-to-client connections for the test?

Reply:
will do